docs/services/sdk/overview.mdx

```diff
@@ -8,23 +8,53 @@ import TabItem from '@theme/TabItem';
 
 # Overview
 
-Restate offers a Typescript SDK to author Restate services using Typescript.
+Restate offers a Java SDK and a Typescript SDK.
+
+## Installation
+
+<Tabs groupId="sdk">
+<TabItem value="ts" label="Typescript">
 
 :::tip
-To get started quickly, use the [Typescript service template generator](https://github.com/restatedev/node-template-generator)!
+To get started quickly, use the [template generator](https://github.com/restatedev/node-template-generator)
 :::
 
-## Installation
 
 Add the `@restatedev/restate-sdk` dependency to your NodeJS project to start developing Restate services.
 
-## API Examples
+:::info
 Restate currently supports two Typescript APIs for implementing services: the Handler API and the gRPC API.
-The gRPC API requires you to specify the service contract (services, methods and data types) via a Protobuf definition, as explained in the [service contract docs](/services/service_type#restate-service-contract).
+The gRPC API requires you to specify the services, methods and data types via a Protobuf definition, as explained in the [service contract docs](/services/service_type#restate-service-contract).
 The Handler API does not require this.
+:::
+
+</TabItem>
+<TabItem value="java" label="Java">
 
-<Tabs groupId="ts-api">
-<TabItem value="handler" label="Handler API Example " default>
+:::tip
+To get started quickly, use the template generator 
+//TODO add link
```

Comment:
Can we at least put the project skeleton in a separate repo and just have `git clone` as a way to get started with a template? For Typescript this is also not in the examples repo, so it would be nice to keep that a bit uniform between languages.